src/components/BlogSnippet/BlogSnippet.tsx

```diff
@@ -0,0 +1,45 @@
+import { getBlogFeed, BlogObj } from '@/api/blog/blog';
+import { formatDate } from '@/helpers/dates';
+import Link from 'next/link';
+import Image from 'next/image';
```

Comment:
We need to use our custom `Image` component in order to remove the `style="color:transparent"` the Next.js component adds to image.